drivers/dai/intel/ssp/ssp.c

```diff
@@ -2555,6 +2557,10 @@ static void ssp_acquire_ip(struct dai_intel_ssp *dp)
 
 		/* Disable dynamic clock gating before touching any register */
 		dai_ssp_pm_runtime_dis_ssp_clk_gating(dp, ssp->ssp_index);
+
+#if SSP_IP_VER >= SSP_IP_VER_2_0
+		sys_write32(ssp->link_clock, dai_i2svss_base(dp) + I2SLCTL_OFFSET);
```

Comment:
Only update the MCLS bits, see line 1994

Reply:
I would also add comment here and in release that the MCLS needs  to be set fro XTAL before power off (reset state)